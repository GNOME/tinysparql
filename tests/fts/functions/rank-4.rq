SELECT ?u { ?u fts:match 'the the' } order by desc fts:rank(?u)
