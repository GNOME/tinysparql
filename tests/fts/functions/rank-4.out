"http://www.example.org/test#10"
"http://www.example.org/test#11"
"http://www.example.org/test#12"
